eq_translations/extractable_strings.py

```diff
@@ -12,6 +12,10 @@
         "json_path": "$.sections[*].repeat.title",
         "description": "Section title (repeating section)",
     },
+    {
+        "json_path": "$.sections[*].repeat.page_title",
+        "description": "Section page_title suffix (repeating section)",
```

Comment:
The text shouldn't  include property names (which aren't exposed to the translator).
```suggestion
        "description": "Section page title suffix (repeating section)",
```